README.md

```diff
@@ -1,54 +1,191 @@
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-Windows users, don't forget to run the [prereq installer](http://github.com/TASVideos/BizHawk-Prereqs/releases) first!
+## Features
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+The BizHawk common features (across all cores) are:
+* rom format and region detection and a corruption check
+* 10 savestate slots and save/load to file
+* speed control, including frame stepping and rewinding
+* memory view/search/edit in all parts of the emulated HW
+* input recording (making TAS movies)
+* take screenshots and record video
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+System | BH Common | Accuracy† | Debugging | Peripherals
+--:|:-:|:-:|:-:|:-:
+Apple II | ✓ | 95%? | ? | ?
+Atari 2600 | ✓ | 95%? | ◕ | ?
+Atari 7800 | ✓ | 95%? | ◕ | ?
+Atari Lynx | ✓ | 95%? | ◔ | ?
+ColecoVision | ✓ | 95%? | ◕ | ?
+Commodore 64 | ✓ | 95%? | ◕ | [some](http://tasvideos.org/Bizhawk/C64.html)
+GB/GBC | ✓ | 95%? | ✓ | ?
+GBA | ✓ | 95%? | ◕ | ?
+Genesis | ✓ | 95%? | ✓ | [all?](https://bitbucket.org/eke/genesis-plus-gx/src/b573cd25853f9f8b5b941fc36506835e228144c6/wiki/Features.md?at=master&fileviewer=file-view-default)
+IntelliVision | ✓ | 95%? | ◕ | ?
+Master System | ✓ | 95%? | ✓ | ?
+N64 | ✓ | 95%? | ◕ | [all?](http://tasvideos.org/Bizhawk/N64.html)
+NGP/NGPC | ✓ | 95%? | ◔ | N/A
+NES | ✓ | 95%? | ✓ | ?
+PSX | ✓ | 95%? | ? | ?
+Saturn | ✓ | 95%? | ◔ | ?
+SNES | ✓ | 95%? | ✓ | ?
+TI-83 | ✓ | 95%? | ? | N/A
+PC Engine / SG | ✓ | 95%? | ✓ | ?
+Virtual Boy | ✓ | 95%? | ? | N/A
+WonderSwan | ✓ | 95%? | ◕ | N/A
+ZX Spectrum | ✓ | 95%? | ✓ | [common](http://tasvideos.org/Bizhawk/ZXSpectrum.html)
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+† *approximately* what portion of games run perfectly, not counting romhacks/homebrew (if you want to count, go right ahead)
 
-## Supported Systems
+EmuHawk's frontend also has a firmware manager, nice overlays, and a comprehensive input mapper. We're talking keyboard and controller mapped to one emulated controller, analog from digital and digital from analog, keyboard hotkeys, controller hotkeys, and rapid fire for emulated controllers. Most of the frontend can be controlled with Lua scripts (only on Windows for now). These scripts can draw overlays for casual play or for RTA speedruns, change the running game to add complex cheats, assist TASers and romhackers in debugging a game, and more. But most importantly: BizHawk is Hawk Biz.
```

Comment:
What is "nice overlays"? 

> We're talking keyboard and controller mapped to one emulated controller, analog from digital and digital from analog, keyboard hotkeys, controller hotkeys, and rapid fire for emulated controllers

 I can't parse this sentence. What are you trying to say?

And Lua seems to belong to BH common.

Reply:
fixed